springexercises/src/main/java/com/spring/exercises/model/User.java

```diff
@@ -0,0 +1,105 @@
+package com.spring.exercises.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private Long id;
+	
+	
+	@Column(nullable = false)
+	private String first_name;
+	
+	@Column(nullable = false)
+	private String last_name;
+	
+	@Column(nullable = false, unique = true)
+	private String email;
+	
+	private String encryptedPassword;
+	
+	private boolean emailVerification;
```

Comment:
pick a case and stick with it for all your fields. You have camelCase and snake_case mixed